app/src/androidTest/test_resources/bike_rental.json

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
On Android, the typical way to bundle these types of raw file resources are under `androidTest/res/raw` - let's use that directory instead.

Reply:
Actually, let's just move these to a JVM build instead - see https://github.com/CUTR-at-USF/opentripplanner-client-library/pull/11/files#r607933744.